content/pl/docs/concepts/overview/_index.md

```diff
@@ -1,7 +1,168 @@
+które mogą być dynamicznie uruchamiane i zarządzane -
+nie jest to monolityczny system działający na jednej, dużej maszynie dedykowanej na wyłączność.
+* Izolacja zasobów: wydajność aplikacji możliwa do przewidzenia
+* Wykorzystanie zasobów: wysoka wydajność i upakowanie.
+
+## Do czego potrzebujesz Kubernetesa i jakie są jego możliwości {#why-you-need-kubernetes-and-what-can-it-do}
+
+Kontenery są dobrą metodą na opakowywanie i uruchamianie aplikacji.
+W środowisku produkcyjnym musisz zarządzać kontenerami, w których działają aplikacje i pilnować, aby nie było żadnych przerw w ich dostępności.
+Przykładowo, kiedy jeden z kontenerów przestaje działać, musi zostać wymieniony.
+Nie byłoby prościej, aby takimi działaniami zajmował się jakiś system?
+
+I tu właśnie przychodzi z pomocą Kubernetes!
+Kubernetes zapewnia środowisko do uruchamiania systemów rozproszonych o wysokiej niezawodności.
+Kubernetes obsługuje skalowanie aplikacji, przełączanie w sytuacjach awaryjnych, różne scenariusze wdrożeń itp.
+Przykładowo, Kubernetes w łatwy sposób może zarządzać wdrożeniem nowej wersji oprogramowania zgodnie z metodyką *canary deployments*.
+
+Kubernetes zapewnia:
+
+* **Detekcję nowych serwisów i balansowanie ruchu**
+Kubernetes może udostępnić kontener używając nazwy DNS lub swojego własnego adresu IP.
+Jeśli ruch przychodzący do kontenera jest duży, Kubernetes może balansować obciążenie i przekierować ruch sieciowy,
+aby zapewnić stabilność całej instalacji.
+* **Zarządzanie obsługą składowania danych**
+Kubernetes umożliwia automatyczne montowanie systemów składowania danych dowolnego typu —
+lokalnych, od dostawców chmurowych i innych.
+* **Automatyczne wdrożenia i wycofywanie zmian**
+Możesz opisać oczekiwany stan instalacji za pomocą Kubernetesa,
+który zajmie się doprowadzeniem w sposób kontrolowany stanu faktycznego do stanu oczekiwanego.
+Przykładowo, przy pomocy Kubernetesa możesz zautomatyzować proces tworzenia nowych kontenerów
+na potrzeby swojego wdrożenia, usuwania istniejących i przejęcia zasobów przez nowe kontenery.
+* **Automatyczne zarządzanie dostępnymi zasobami**
+Twoim zadaniem jest dostarczenie klastra maszyn, które Kubernetes może wykorzystać do uruchamiania zadań w kontenerach.
+Określasz zapotrzebowanie na moc procesora i pamięć RAM dla każdego z kontenerów.
+Kubernetes rozmieszcza kontenery na maszynach w taki sposób, aby jak najlepiej wykorzystać dostarczone zasoby.
+* **Samoczynne naprawianie**
+Kubernetes restartuje kontenery, które przestały działać, wymienia je na nowe, wymusza wyłączenie kontenerów,
+które nie odpowiadają na określone zapytania o stan
+i nie rozgłasza powiadomień o ich dostępności tak długo, dopóki nie są gotowe do działania.
+* **Zarządzanie informacjami poufnymi i konfiguracją**
+Kubernetes pozwala składować i zarządzać informacjami poufnymi, takimi jak hasła, tokeny OAuth i klucze SSH.
```

Comment:
... czy certyfikaty.

Reply:
W oryginale tylko to, zmieniłem trochę, żeby nie sugerowało zamkniętej listy.